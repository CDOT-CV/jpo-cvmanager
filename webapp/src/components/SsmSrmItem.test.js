import React from "react";
import { render } from "@testing-library/react";
import SsmSrmItem from "./SsmSrmItem";

it("should take a snapshot", () => {
  const { asFragment } = render(<SsmSrmItem elem={{}} setSelectedSrm={() => {}} />);

  expect(asFragment(<SsmSrmItem elem={{}} setSelectedSrm={() => {}} />)).toMatchSnapshot();
});
