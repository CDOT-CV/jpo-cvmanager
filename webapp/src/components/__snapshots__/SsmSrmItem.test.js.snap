// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    id="ssmitemdiv"
  >
    <p
      class="ssmsrmitemtext"
    />
    <p
      class="ssmsrmitemtext"
    />
    <p
      class="ssmsrmitemtext"
    />
    <p
      class="ssmsrmitemtext"
    />
    <button
      class="btnDisabled"
      disabled=""
    >
      View
    </button>
  </div>
</DocumentFragment>
`;
